x/committee/msg_server_test.go

```diff
@@ -0,0 +1,208 @@
+package committee_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	proposal "github.com/cosmos/cosmos-sdk/x/params/types/proposal"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	"github.com/kava-labs/kava/app"
+	"github.com/kava-labs/kava/x/committee"
+	"github.com/kava-labs/kava/x/committee/keeper"
+	"github.com/kava-labs/kava/x/committee/types"
+	swaptypes "github.com/kava-labs/kava/x/swap/types"
+)
+
+//var _ types.PubProposal = UnregisteredPubProposal{}
+//
+//// UnregisteredPubProposal is a pubproposal type that is not registered on the amino codec.
+//type UnregisteredPubProposal struct{}
+//
+//func (UnregisteredPubProposal) GetTitle() string       { return "unregistered" }
+//func (UnregisteredPubProposal) GetDescription() string { return "unregistered" }
+//func (UnregisteredPubProposal) ProposalRoute() string  { return "unregistered" }
+//func (UnregisteredPubProposal) ProposalType() string   { return "unregistered" }
+//func (UnregisteredPubProposal) ValidateBasic() error   { return nil }
+//func (UnregisteredPubProposal) String() string         { return "unregistered" }
+
+// NewDistributionGenesisWithPool creates a default distribution genesis state with some coins in the community pool.
+//func NewDistributionGenesisWithPool(communityPoolCoins sdk.Coins) app.GenesisState {
+//gs := distribution.DefaultGenesisState()
+//gs.FeePool = distribution.FeePool{CommunityPool: sdk.NewDecCoinsFromCoins(communityPoolCoins...)}
+//return app.GenesisState{distribution.ModuleName: distribution.ModuleCdc.MustMarshalJSON(gs)}
+//}
+
+type MsgServerTestSuite struct {
```

Comment:
These tests are based on the previous handler tests -- they can probably be improved as not all proposal types and some error cases are not handled